Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -972,6 +972,8 @@ namespace Babylon
                         const auto referenceSpaceType = info[0].As<Napi::String>().Utf8Value();
                         assert(referenceSpaceType == XRReferenceSpaceType::UNBOUNDED ||
                             referenceSpaceType == XRReferenceSpaceType::VIEWER);
+                        (void)XRReferenceSpaceType::UNBOUNDED;
+                        (void)XRReferenceSpaceType::VIEWER;
```

Comment:
Is this to prevent these variables from getting compiled out? If so, and if this is the way we actually want to do this, can you specify that in a comment just for clarity's sake?

Reply:
Actually, we need some UNUSED(x) macro. It's on my todo list.